HLTrigger/HLTcore/interface/HLTPrescaleProvider.h

```diff
@@ -79,7 +79,7 @@ class HLTPrescaleProvider {
   // any one negative => error in retrieving this (L1T or HLT) prescale
 
   // In case of a complex Boolean expression as L1 seed
-  template <typename TL1 = int, typename THLT = TL1>
+  template <typename TL1 = double, typename THLT = TL1>
```

Comment:
so the plan here was to make this  a breaking change and not to silently update to the new values. 

The issue that client code (which we have no control of) may not know to update. Thus we have to alert them somehow. 

If we do this, they will suddenly start getting the double function. But they are still using ints so have a bug. 

They will need to do prescaleValues<double>()  ie change their function signature so they know there is a change. 

When we want to do this, we put a template specialisation for int which throws a static_assert (or similar) so that existing code has to be updated.

So summary, please do not change the default here to a double, it'll cause problems

Reply:
Isn't it already past time to drop the `int` and `unsigned int` cases or replace them with an error ?